pom.xml

```diff
@@ -491,7 +514,7 @@
         <artifactId>flatten-maven-plugin</artifactId>
         <version>1.1.0</version>
         <configuration>
-          <updatePomFile>true</updatePomFile>
+          <updatePomFile>false</updatePomFile>
```

Comment:
looking at this more locally, I think setting this to false also made it so the plugin didn't put an updated pom into the repo as a result of running `install`. It also looks like after the changes to wrap the maven plugin for yetus we don't generate maven commands that cause the flatten plugin to fail with this set to true.